swatch-core/src/main/java/org/candlepin/subscriptions/db/OfferingRepository.java

```diff
@@ -36,6 +37,10 @@ public interface OfferingRepository extends JpaRepository<Offering, String> {
 
   List<Offering> findBySkuIn(Set<String> skus);
 
+  @EntityGraph(value = "graph.offering")
+  @Query(value = "select o from Offering o where o.sku = :sku")
```

Comment:
Please add an `order by` clause here.  See https://hibernate.atlassian.net/browse/HHH-17040 and https://github.com/RedHatInsights/rhsm-subscriptions/pull/2383